sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -264,6 +264,14 @@ public void recognizeEntitiesForListWithOptions() {
                 .verifyComplete());
     }
 
+    @Test
+    public void recognizeEntitiesTooManyDocuments() {
```

Comment:
although this will be supported in all endpoints, it has only been deployed to linked entities one.

Reply:
ohh wait, does it works now in the entities endpoint too?

Reply:
only recognizedEntltiy and recognizedLinkedEntity works as i tested out.